receiver/dockerstatsreceiver/metadata.yaml

```diff
@@ -628,3 +628,19 @@ metrics:
       aggregation: cumulative
     attributes:
       - interface
+
+  # Pids
+  container.pids.count:
+    enabled: false
+    description: "Number of pids in the container's cgroup."
+    extended_documentation: "It requires docker API 1.23 or higher and kernel version >= 4.3 with pids cgroup supporeted. [More docs](https://www.kernel.org/doc/Documentation/cgroup-v1/pids.txt)"
+    unit: "{pids}"
+    gauge:
```

Comment:
I believe these new metrics should be an up down counter (non-monotonic sum) instead of a gauge. Similar to many of the memory metrics elsewhere in this component. 

This is according to the semantic convention spec for similar metric `system.processes.count`: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemprocesses---aggregate-system-process-metrics